frontend/routes/viewerGui/components/previewDetails/previewDetails.component.tsx

```diff
@@ -88,23 +88,16 @@ export class PreviewDetails extends React.PureComponent<IProps, any> {
 	public textFieldRef = React.createRef<any>();
 	public scrollableContainerRef = React.createRef<HTMLDivElement>();
 
-	public renderNameWithCounter = renderWhenTrue(() => {
-		const title = `${this.props.number}. ${this.props.name}`;
-		return (
-			<Tooltip title={title}>
-				<Typography paragraph>
-					{title}
-				</Typography>
-			</Tooltip>
-		);
-	});
+	public renderNameWithCounter = renderWhenTrue(() => (
+		<Typography paragraph>
```

Comment:
Is there a reason why tooltips have been removed?

Reply:
I think this happened by getting latest ? 😕

Reply:
or if I did it its by mistake